### PR TITLE
Update reserves info

### PR DESCRIPTION
**Pull request by mDuo13**
Following up on #2885:

- Add a blog post about lower reserves
- Updated variables in the env file with updated reserve settings
- Update many mentions of the "current" reserve settings to use the env variables (so that this will be fewer changes next time) including in Japanese and Spanish translations.
- Add comments with the string `RESERVES_REMINDER` to places where the documentation includes math or other values based on reserve amounts that need to be manually updated when reserves change. Hopefully this should make it easier to find them all next time.

### CODE REVIEW

**Review by amarantha-k — DISMISSED** [review]
The date on the blog post needs to be updated. Otherwise, updates look good.

File: blog/2024/lower-reserves-are-in-effect.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+markdown:
+    editPage:
+        hide: true
+date: 2024-12-06
+labels:
+    - Advisories
+---
+# Lower Reserves Are In Effect
+
+This week, the reserve requirements for using the XRP Ledger blockchain changed by the collective voting of the validators. The new requirements are **1 XRP** base reserve per account (down from 10 XRP), and **0.2 XRP** owner reserve increment per item (down from 2 XRP).
```
Comment (amarantha-k) [inline blog/2024/lower-reserves-are-in-effect.md]: Use the date instead?

File: blog/2024/lower-reserves-are-in-effect.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+category: 2024
+markdown:
+    editPage:
+        hide: true
+date: 2024-12-06
```
Comment (amarantha-k) [inline blog/2024/lower-reserves-are-in-effect.md]:
```suggestion
date: 2024-12-11
```

**Comment by mDuo13** [comment]
Rebased on master to fix merge conflicts and squash edits into the same commit.

**Review by DennisDawson — APPROVED** [review]
LGTM

**Pull request merged**